packages/core/package.json

```diff
@@ -27,10 +27,10 @@
     "node": ">=10"
   },
   "peerDependencies": {
-    "redux": "^4.0.5"
+    "redux": "*"
```

Comment:
Are we compatible with all versions of redux?

Reply:
We're not, ill fix this asap